prac_05/hex_colours.py

```diff
@@ -0,0 +1,17 @@
+"""
+CP1404 - Prac 5
+Program: Hex Colours
+"""
+
+COLOUR_TO_CODE = {"Amethyst": "#9966cc", "Aqua": "#00ffff", "Beige": "#f5f5dc", "Black": "#000000",
```

Comment:
The dictionary COLOUR_TO_CODE is appropriately named in uppercase to indicate it’s a constant.